src/Console/Helper/ComposerHelper.php

```diff
@@ -45,26 +44,20 @@ public function getRootPackage()
         return $this->rootPackage;
     }
 
-    /**
-     * @return bool
-     */
-    public function hasRootPackage()
+    public function hasRootPackage(): bool
     {
         return null !== $this->rootPackage;
     }
 
     /**
-     * @return Config|null
+     * @return Config|null|array
```

Comment:
Type should be `?Config`. Not sure why array was added, but it should never be just array